ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm

```diff
@@ -69,3 +69,102 @@
 	new /obj/item/radio/headset/syndicate/alt(src)
 	new /obj/item/clothing/gloves/combat(src) //1 TC?
 	new /obj/item/gun/ballistic/automatic/pistol(src) //7 TC
+
+// Advanced hypno-flash. For psychologists.
+/obj/item/assembly/flash/hypnotic/adv
+	desc = "A modified flash device, used to beam preprogrammed orders directly into the target's mind."
+	light_color = COLOR_RED_LIGHT
+	var/hypno_cooldown = 29 SECONDS // Big damn cooldown
```

Comment:
Is the flash's cooldown really worth making it its own item? The other flash can just let you sleepy pen them with hallucinagens, or you can just put them to sleep

Reply:
You can, but this one is instant, doesn't require you to waste your precious time forcing them to sleep.
I'll also increase cooldown to 59 seconds, to avoid "revolutions".